incompatibility-allowlist

```diff
@@ -18,3 +18,8 @@ firefox-accounts.account-ecosystem.*
 # This commit needs to be reverted due to the schema being incorrect.
 # https://github.com/mozilla-services/mozilla-pipeline-schemas/commit/6172d894f37b4330aca87a1d8a5ff5acf85b4206
 # rally-zero-one.*
+
+# Bug 1706875
+# The rally-debug schema was commit using the structured metadata field instead
+# of the pioneer metadata schemas.
+rally-debug.*
```

Comment:
Actually, I'm not sure this is strictly necessary with my test run, but it's more of a precaution than anything.